src/LiteCore/src/LiteCore.Shared/Interop/C4Replicator_native.cs

```diff
@@ -45,11 +45,32 @@ internal unsafe static partial class Native
         [DllImport(Constants.DllName, CallingConvention = CallingConvention.Cdecl)]
         public static extern C4ReplicatorStatus c4repl_getStatus(C4Replicator* repl);
 
+        public static byte[] c4repl_getPendingDocIDs(C4Replicator* repl, C4Error* outErr)
+        {
+            using(var retVal = NativeRaw.c4repl_getPendingDocIDs(repl, outErr)) {
+                return ((FLSlice)retVal).ToArrayFast();
```

Comment:
what if `retVal` is nil from lite-core?

Reply:
This class is actually C# binding class... Auto generated.

Reply:
https://github.com/couchbase/couchbase-lite-core/commit/82df7b625b23b0540bad52e8667e304b78b83f10#diff-97c2a0f6c7da83884c12f2f63f627122R244 <--- possibly from here

Reply:
By the way @jayahariv `FLSlice` and `FLSliceResult` cannot be null since they are value types.